packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -8,6 +8,7 @@ import { WalletText } from '../WalletText';
 import './WalletDropdown.scss';
 
 type TProps = {
+    helperMessage?: string;
```

Comment:
whats is helper message? just wondering if its within walletdropdown responsibility

Reply:
Its message under the dropdown. We will use WalletTextField inside the dropdown maybe. Removing this for now